src/snyk/common/configuration/configuration.ts

```diff
@@ -498,10 +486,6 @@ export class Configuration implements IConfiguration {
 
   private getConfigName = (setting: string) => setting.replace(`${CONFIGURATION_IDENTIFIER}.`, '');
 
-  private static isSingleTenant(url: URL): boolean {
```

Comment:
This check is only for when the custom endpoint is provided, which is now only going to be `api.*.snyk.io`

Reply:
Just realized: this may be why fedramp is having problems right now - we need `snykgov.io` as valid domain :). But yeah, I think I need to look deeper into the code - settings and how it flows together.